.circleci/config.yml

```diff
@@ -42,6 +42,17 @@ jobs:
             else
               echo "build is skipped ..."
             fi
+      - run:
+          name: download all maven dependencies and groovy (second try)
+          command: |
+            RUN_JOB=`cat .ci-temp/run_job`
+            echo "RUN_JOB="$RUN_JOB
+            if [[ $RUN_JOB == 1 ]]; then
```

Comment:
Is status always 1?
May be it is better to make it like ` $RUN_JOB != 0` ?

Reply:
I do not like negations, positive conditions works much better  and less error prone.

Reply:
I use this condition to quickly disable execution of CI at all, when I do debuging on remote(in CI) for certain CI. So I disable all except for one CI and and push very very frequently to debug by `echo`.